app/main.py

```diff
@@ -1,34 +1,80 @@
+from __future__ import annotations
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
```

Comment:
Place all arguments in the `__init__` on a new line including `self` to improve the code readability and perception.